serialization/ksp_plugin.proto

```diff
@@ -107,6 +108,9 @@ message Plugin {
 message Vessel {
   required MasslessBody body = 1;
   optional FlightPlan flight_plan = 4;
+  optional Trajectory history = 6;  // required
```

Comment:
No point in putting the history in the source code, it's much easier to obtain it through "blame".  However, it's important to state that even though this field is "optional" (for compatibility) it really is required in new code.  We have comments like that everywhere.